asm-to-pil/src/lib.rs

```diff
@@ -23,6 +23,9 @@ pub fn compile<T: FieldElement>(file: AnalysisASMFile) -> AnalysisASMFile {
                         }
                         Item::Expression(e) => Item::Expression(e),
                         Item::TypeDeclaration(enum_decl) => Item::TypeDeclaration(enum_decl),
+                        Item::TraitImplementation(trait_impl) => {
+                            Item::TraitImplementation(trait_impl)
+                        }
                         Item::TraitDeclaration(trait_decl) => Item::TraitDeclaration(trait_decl),
```

Comment:
Can you use something like
```suggestion
                        Item::Expression(_) | Item::TypeDeclaration(_) | ... => m
```
?

Reply:
Maybe we could also just match on `Machine` and only modify the machine in that case, and pass the rest un-modified.

Reply:
Could also use the `machines_mut` function